apps/dock/dock.talon

```diff
@@ -0,0 +1,5 @@
+os: mac
+-
+^desktop$: user.dock_send_notification("com.apple.showdesktop.awake")
+^window$: user.dock_send_notification("com.apple.expose.front.awake")
```

Comment:
Apparently this stopped working since I implemented it. Does it work for you? I tested both on macOS 14 and 15 and on neither OS did it do anything.

Don't think this is a barrier to merging though!

Reply:
Ah you're right, I just poked around and it looks like it's now `com.apple.expose.awake` (without `front`).  I put that in a PR here, mind taking a look?

https://github.com/talonhub/community/pull/1608